core/src/main/scala/chisel3/VerificationStatement.scala

```diff
@@ -299,13 +449,11 @@ private object VerificationStatement {
     (p.source.file.name, p.line, p.lineContent.trim)
   }
 
-  // creates a printf to inform the user of a failed assertion or assumption
   def failureMessage(
     kind:     String,
     lineInfo: SourceLineInfo,
     cond:     Bool,
-    message:  Option[String],
-    data:     Seq[Bits]
+    message:  Option[Printable]
```

Comment:
This is a binary incompatible change, you need the old one (deprecate it)

Reply:
Actually this is part of a private object, it will show up as a binary incompatibility but since users can't call it, it doesn't matter.

Reply:
So @jackkoenig  we are good with what we have? No need to add the old one right?

Reply:
Correct, sorry for the confusion. I had checked binary compatibility and it flagged this, but there are things in Scala that have to be implemented in Java as public APIs even though there private from Scala's perspective and are thus "false positives" if that makes sense.